### DISCUSSION TITLE
Anthropic Example is throwing a TypeError: Messages.create() got an unexpected keyword argument 'tools'

### DISCUSSION THREAD

**Discussion by edmar** [567-labs/instructor#807] [Q&A] [answered]
- [x] This is actually a bug report.
- [ ] I am not getting good LLM Results
- [ ] I have tried asking for help in the community on discord or discussions and have not received a response.
- [ ] I have tried searching the documentation and have not found an answer.

**What Model are you using?**

- [ ] gpt-3.5-turbo
- [ ] gpt-4-turbo
- [ ] gpt-4
- [x] Claude 

**Describe the bug**
The provided Anthropic example is not working. When you run it there is an error:

TypeError: Messages.create() got an unexpected keyword argument 'tools'

**To Reproduce**
Run the code from https://github.com/jxnl/instructor/blob/main/examples/anthropic/run.py

**Expected behavior**

To get a user to be returned.

**Comment by FirstGalacticEmpire** [comment]
I'm experiencing very similar issue. I thinks Claude is called with kwarg tools, which is incorrect.

**Comment by ar-mccabe** [comment]
+1, started noticing this error this morning with `claude-sonnet-3-5-...` and `instructor.from_anthropic(Anthropic)` when calling `.chat.completions.create_with_completion(...)`

**Comment by JakeBeresford** [comment, marked as answer]
We are also seeing this error. Seems related to #723 - if I set `mode=instructor.mode.Mode.ANTHROPIC_JSON` when initializing the client, we do not have this error.

**Comment by edmar** [comment]
I have just updated to 1.3.4 and now it seems that anthropic integration is working.

**Comment by arcaputo3** [comment]
Setting `anthropic = "^0.30.0"` avoids the tools issue, but appears to run into serialization issues with, for example, `datetime` fields. Using `mode=instructor.mode.Mode.ANTHROPIC_JSON` seems to be most reliable for us at this point.
